src/util/generatePreview.js

```diff
@@ -38,6 +39,12 @@ const errorHandlerScript = `(${(() => {
   };
 }).toString()}());`;
 
+const alertReplacementScript = `(${(() => {
+  window.alert = (message) => {
+    swal(message); // eslint-disable-line no-undef
```

Comment:
Also, is it worth doing a `delete window.swal` once we have bound it to `alert`? Would be nice to have the environment the students’ code is running in be as pristine as possible.